tests/base/entities/test_celltype.py

```diff
@@ -20,3 +20,9 @@ def test_cl_celltype_inspect_name():
     expected_series = pd.Series([True, True, True, True, False])
 
     assert inspect.equals(expected_series)
+
+
+def test_cl_celltype_version():
+    # newer version, not yet in s3://bionty-assets
```

Comment:
As we might add that version to bionty-assets later, it might make sense to test an old version that we also don't have on s3.